hdfs/src/main/java/io/ddf/hdfs/HDFSDDFManager.java

```diff
@@ -68,31 +70,24 @@ public DataFormat getDataFormat(HDFSDDF hdfsDDF) throws DDFException {
     if (hdfsDDF.getIsDir()) {
       try {
         FileStatus[] files = this.fs.listStatus(new Path(hdfsDDF.getPath()));
-        if (files.length == 0) {
-          throw new DDFException(String.format("There is no file under %s", hdfsDDF.getPath()));
-        }
+        // if (files.length == 0) {
```

Comment:
I kept it because i think here the behavior of empty folder may still change. And it's better to keep it here as a reminder/note.